lib/services/user-service.js

```diff
@@ -0,0 +1,65 @@
+var fs = require('fs');
```

Comment:
the testing-cloud-app is relatively simple and there is no service layer yet, so I'm not sure if it is worthwhile to introduce it, moreover in this code is lot of duplication with db.js

I would prefer to change/refactor db.js so the code for creating random user will stay in one place